src/Parallel/GlobalCache.hpp

```diff
@@ -461,6 +463,28 @@ GlobalCache<Metavariables>::get_self_proxy() noexcept {
   return this->thisProxy;
 }
 
+#if defined(__GNUC__) && !defined(__clang__)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wsuggest-attribute=noreturn"
```

Comment:
really? 😞

Reply:
yeah... even though the error is in the if... -.-